IHP/Test/Database.hs

```diff
@@ -34,8 +38,11 @@ createTestDatabase databaseUrl = do
             |> cs
 
     libDir <- LibDir.findLibDirectory
-    importSql newUrl (cs libDir <> "/IHPSchema.sql")
-    importSql newUrl "Application/Schema.sql"
+
+    let importSql file = Process.callCommand ("psql " <> (cs newUrl) <> " < " <> file)
```

Comment:
Maybe we should also add a comment why `psql` is used here isntead of the old approach. Otherwise we might change that back in the future because someone doesn't know about the encoding probelms :D